Pico/Motors.cpp

```diff
@@ -1,38 +1,35 @@
 #include "motors.h"
 #include "Arduino.h"
 
-Motors::Motors(uint8_t speed1, uint8_t in1_1, uint8_t in2_1, uint8_t speed2, uint8_t in1_2, uint8_t in2_2, uint8_t speed3, uint8_t in1_3, uint8_t in2_3, uint8_t speed4, uint8_t in1_4, uint8_t in2_4)
-    : motor1(speed1, in1_1, in2_1),
-      motor2(speed2, in1_2, in2_2),
-      motor3(speed3, in1_3, in2_3),
-      motor4(speed4, in1_4, in2_4){};
-
-void Motors::InitializeMotors()
-{
+Motors::Motors(uint8_t speed1, uint8_t in1_1, uint8_t in2_1, uint8_t speed2, uint8_t in1_2, uint8_t in2_2, uint8_t speed3, uint8_t in1_3, uint8_t in2_3, uint8_t speed4, uint8_t in1_4, uint8_t in2_4) 
+: motor1(speed1, in1_1, in2_1),
+  motor2(speed2, in1_2, in2_2),
+  motor3(speed3, in1_3, in2_3),
+  motor4(speed4, in1_4, in2_4)
+{};
```

Comment:
Would it be possible to import the `constants.h` file here?
And instead of passing the pins as parameters from the `.ino` file, you directly get them from the `constants`. For example:
```cpp
Motors::Motors():
    motor1( MOTOR4_PWM, MOTOR4_IN1, MOTOR4_IN2),
    ....
```
This way, when declaring the `Motors` object in the `.ino` file, it looks cleaner.